src/tree_cmd.cpp

```diff
@@ -193,11 +220,20 @@ static void PlaceTreeGroups(uint num_groups)
 			int x = GB(r, 0, 5) - 16;
 			int y = GB(r, 8, 5) - 16;
 			uint dist = abs(x) + abs(y);
+
+			uint max_dist = MAX_DISTANCE_FROM_CENTER;
+
+			if (_settings_newgame.game_creation.tree_placer == TP_IMPROVED)
+				max_dist = GB(r, 16, 3) + 8;
```

Comment:
Exactly what @michicc said; coding style, please never put a if statement with a single line body over two lines without {}. So either:

```if (_settings_newgame.game_creation.tree_placer == TP_IMPROVED) max_dist = GB(r, 16, 3) + 8;```

or

```
if (_settings_newgame.game_creation.tree_placer == TP_IMPROVED) {
    max_dist = GB(r, 16, 3) + 8;
}
```

Same goes for the line below :)
This to prevent future errors, where someone mistakes the indentation for a start of a new scope.

Reply:
Oh, ok, got it!